.github/workflows/build.yml

```diff
@@ -63,51 +63,18 @@ jobs:
       - run: pip3 install -r drakrun/requirements.txt
       - name: Check with flake8
         run: flake8 --extend-ignore=E501,E203 --max-line-length=88 drakrun/
-  package_drakcore:
```

Comment:
would be nice if we could split this file into smaller ones, if not then so be it

Reply:
Yeah, I guess we can use composite actions for that

Reply:
On the other hand: these actions strongly depend on each other so it's not easy to split them. I would leave it as is.